web/app/src/ui/games-list/GameCard.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import {Game, getGameGreenPlayer, getGameRedPlayer} from "../../model/game";
```

Comment:
pro tip: we usually put imports for outside packages first, and imports for local scripts second. It's just a convention, but I find it useful to immediately see external deps.